hg-fast-export.py

```diff
@@ -359,7 +364,7 @@ def verify_heads(ui,repo,cache,force,branchesmap):
 
 def hg2git(repourl,m,marksfile,mappingfile,headsfile,tipfile,
            authors={},branchesmap={},tagsmap={},
-           sob=False,force=False,hgtags=False,notes=False,encoding='',fn_encoding=''):
+           sob=False,force=False,hgtags=False,notes=False,encoding='',fn_encoding='',commitmsg=False):
```

Comment:
The name of the parameter does not match its purpose, from the name I could never guess that it is controlling the addition of a prefix to the title line of the commit message.